editor/inserter/menu.js

```diff
@@ -19,6 +19,8 @@ import { getCategories, getBlockTypes } from 'blocks';
 import './style.scss';
 import { showInsertionPoint, hideInsertionPoint } from '../actions';
 
+const debouncedSpeak = debounce( wp.a11y.speak, 500 );
```

Comment:
I see you made some adjustments based on https://github.com/WordPress/gutenberg/pull/1508#discussion_r124771019.

There I said that it might be better to move inside the constructor, not outside the class. :)